Makefile

```diff
@@ -1,7 +1,7 @@
-PREFIX ?=	$PREFIX
+include .config
```

Comment:
Well, making something generic seems like a good idea.

The point to having a separate configuration file to be included by the makefile(s) is to make verification simpler, in case any other bug shows up.